E.Deezer/Api/GenreWithRadios.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace E.Deezer.Api
+{
+    public class GenreWithRadios : IGenreWithRadios
```

Comment:
Not sure about this class name. If this is just a response class should probably have this in it's name. 

Also! Class should be private

Reply:
Yes, i agree with You this is not the best name but i wasn't able to come up with a better one. Do You have any idea how it should be called? This is a response class, like the `Artist`, `Genre`, etc.